backend/app/src/game-socket/GameSocketEvents.ts

```diff
@@ -0,0 +1,44 @@
+
+import
+{
+	OnGatewayConnection,
+	OnGatewayDisconnect,
+	OnGatewayInit,
+	WebSocketGateway,
+	WebSocketServer
+} from "@nestjs/websockets";
+import { Server, Socket } from "socket.io";
+
+
+@WebSocketGateway(
+{
+	cors:
+	{
+		origin: "*"
+	},
+	// namespace: "/game-engine-v2d"
```

Comment:
This one is commented, for now, not in the scope of the issue. 
Meeting requiered.